anchor-service/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
Can you add the `/anchor-service/` dir to the CODEOWNERS file so we establish reviewers for the rest of this code.